File: codes/1930.js
function calc(values){
  let tomadas = values[0].split(' ');
  let equip = 1;
  tomadas.forEach(tomada => {
    equip += tomada -1;
  })
  console.log(equip)
}
calc(["2 4 3 2"])
calc(["6 6 6 6"])
calc(["2 2 2 2"])

File: codes/1930.py
#Não finalizado
def calc(values):
  tomadas = values[0].split(' ')
  equip = 1
  equip += [int(tomada) - 1 for tomada in tomadas]
#   for tomada in tomadas:
#     equip += int(tomada) - 1
  
  print(equip)


calc(["2 4 3 2"])
# calc(["6 6 6 6"])
# calc(["2 2 2 2"])
